src/test/kotlin/com/atlassian/performance/tools/virtualusers/lib/infrastructure/FixedPublicJiraSoftwareDistribution.kt

```diff
@@ -0,0 +1,27 @@
+package com.atlassian.performance.tools.virtualusers.lib.infrastructure
+
+import com.atlassian.performance.tools.infrastructure.api.distribution.ProductDistribution
+import com.atlassian.performance.tools.infrastructure.api.distribution.PublicJiraSoftwareDistribution
+import com.atlassian.performance.tools.ssh.api.SshConnection
+import java.net.URI
+import java.time.Duration
+
+/**
+ * Works around a bug in [PublicJiraSoftwareDistribution]:
+ * ```
+ * pget: /software/jira/downloads/atlassian-jira-software-7.13.0.tar.gz: Cannot assign requested address
```

Comment:
Is the bug tracked? A ticket link would be nice we would know when to remove this workaround.